xcode/Subconscious/Shared/Components/Detail/Detail.swift

```diff
@@ -1659,7 +1659,7 @@ struct DetailModel: ModelProtocol {
             through: { markup in
                 switch markup {
                 case .slashlink(let slashlink):
-                    let replacement = link.address.slug.toSlashlink()
+                    let replacement = link.address.slug.markup
```

Comment:
Slashlink is now an actual type. Previously we used this term to mean "markup text of a slug". `.toSlashlink()` now returns the actual type `Slashlink`. The previous functionality is supplied via `.markup`.